src/fix/main.c

```diff
@@ -68,6 +68,80 @@ static void print_usage(void)
 	exit(1);
 }
 
+static long parse_cartridge(char *arg, char **ep)
+{
+	long cartridge = -1;
+
+	if (!arg || *arg == '\0' || *arg == '+' || *arg == '-' ||
+		(*arg >= '0' && *arg <= '9'))
+		return strtoul(arg, ep, 0);
+
+	*ep = arg + strlen(arg);
+
+	/* cartridge type names from Pan Docs; none of them start with [-+0-9] */
+	/* https://gbdev.io/pandocs/#_0147-cartridge-type */
+	if (!strcmp(arg, "ROM ONLY"))
```

Comment:
This one is annoying to pass as a command line argument because of the space.

Reply:
How about allowing `"ROMONLY"` or `"ROM ONLY"`? Likewise for `"POCKET CAMERA"` and `"BANDAI TAMA5"`.

Reply:
Why not simply `"NONE"`=

Reply:
The names are as listed in Pan Docs.

Reply:
Maybe we could support some alternatives for convenience (e.g. `ROM`, `NONE`)?

Reply:
``ROM``, ``CAMERA`` and ``TAMA5`` look like good alternatives for me, if we are trying to keep them consistent.

Reply:
Actually, why don't we just replace the spaces by underscores or dashes?

Reply:
I've added aliases with underscores (e.g. `"POCKET CAMERA"` and `"POCKET_CAMERA"` both work) and `"NONE"` for `"ROM ONLY"`/`"ROM_ONLY"`.